AppCenterDataStorage/AppCenterDataStorage/MSDocumentWrapper.h

```diff
@@ -14,21 +14,57 @@
  */
 @property(nonatomic, strong, readonly) T deserializedValue;
 
+/**
+ * Partition.
```

Comment:
Can you please add some more detail about what "Partition" means in this context?